src/carbonio-files-ui-common/views/components/List.preview.test.tsx

```diff
@@ -70,4 +70,44 @@ describe('Preview action', () => {
 		expect(screen.queryByText(/This item cannot be displayed/i)).not.toBeInTheDocument();
 		expect(screen.queryByRole('button', { name: /download file/i })).not.toBeInTheDocument();
 	});
+
+	test.each([
+		['image/png'],
+		['application/vnd.openxmlformats-officedocument.wordprocessingml.document'],
+		['application/vnd.oasis.opendocument.presentation'],
+		['application/vnd.oasis.opendocument.spreadsheet'],
+		['application/vnd.oasis.opendocument.text'],
+		['application/vnd.openxmlformats-officedocument.presentationml.presentation'],
+		['application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'],
+		['application/pdf']
+	])('Check if the context preview action is supported by mime type', async (mimeType) => {
```

Comment:
Now these are the mime types which have the action, but thinking about the original test you made (before the split), there should be also a test to check that the action is not visible for mime types different from these ones